lib/committee/query_hash_coercer.rb

```diff
@@ -0,0 +1,35 @@
+module Committee
+  class QueryHashCoercer
```

Comment:
I've done a very poor job of this elsewhere in the library, but would you mind adding a short description on this class indicating exactly what it's for?